src/main/java/jsr223/docker/compose/utils/CommandlineOptionsFromBindingsExtractor.java

```diff
@@ -53,6 +54,12 @@
 
     public final static String DOCKER_COMPOSE_COMMANDLINE_OPTIONS_SPLIT_REGEX_DEFAULT = " ";
 
+    public final static String DOCKER_FILE_COMMANDLINE_OPTIONS_SPLIT_REGEX_DEFAULT = " ";
```

Comment:
You can remove `public final static String DOCKER_COMPOSE_UP_COMMANDLINE_OPTIONS_KEY = "docker-compose-up-options";`, `public final static String DOCKER_COMPOSE_COMMANDLINE_OPTIONS_KEY = "docker-compose-options";`, `public final static String DOCKER_COMPOSE_COMMANDLINE_OPTIONS_KEY = "docker-compose-options";`, `public final static String DOCKER_COMPOSE_COMMANDLINE_OPTIONS_SPLIT_REGEX_KEY = "docker-compose-options-split-regex";`

Reply:
And methods which are connected with it. Or if we need the same functionality with the docker file command then you can keep it but rename it.

Reply:
This file is shared by the two script engines